packages/babel-parser/test/fixtures/typescript/types/typeof-private-error/input.ts

```diff
@@ -0,0 +1,11 @@
+class C {
+    #a = 'a';
+
+    constructor() {
+        const a: typeof this.#a = ''; // Ok
+        const b: typeof this.#a = 1;  // Error
```

Comment:
I was initially confused by this comment, then realized that it's a type error and not a syntax error that `@babel/parser` would throw. Could you remove the comment. 🙏

Reply:
Sure.
I copied it from the TS test case.
To avoid confusion, it should be removed.